README.md

```diff
@@ -80,6 +81,7 @@
 39. [**10minutemail.com**](http://10minutemail.com/10MinuteMail/index.html)- To avoid spam create a temporary email.
 40. [**codepen.io**](http://codepen.io/)- Edit HTML,CSS,JS online. Playground for front end web development.
 41. [**Zeta Uploader**](https://www.zeta-uploader.com/)- Zeta Uploader is used to send large files to any email address without any login/signup and you can protect your uploaded or sent data by password. For more detail visit Zeta Upolader site. 
+42. [**web.whatsapp.com**](https://www.whatsapp.com)- To connect whatsapp in mobile with computer. Also shows a pop-up notification whenever a message is recieved.
```

Comment:
@Akarshit tell me what exactly should I edit. Should I add Franz instead of web.whatsapp.com? Should I keep the team viewer as it is or remove it?